js/libs/core.js

```diff
@@ -617,39 +625,72 @@ Fliplet.FormBuilder = (function() {
       var hasOptions = component.props.options && Array.isArray(component.props.options.type());
       var hasSelectAll = component.props.addSelectAll && typeof component.props.addSelectAll.default === 'boolean';
       var isSlider = component.props._componentName.default === 'flSlider';
+      var isMatrixField = component.props._componentName.default === 'flMatrix';
```

Comment:
`isSlider` in the line above doesn't have the word `field` in the variable name. Why should this have the `field` word in the variable name?

```suggestion
      var isMatrix = component.props._componentName.default === 'flMatrix';
```

Reply:
did the changes